### DISCUSSION TITLE
Provide profiles for statistics collection - different keyboards in mind

### DISCUSSION THREAD

**Discussion by Comradin** [monkeytypegame/monkeytype#4910] [Ideas] [answered]
I have different keyboards and with some (split) I am slower with and make more mistakes as with others.

As a registered user the site collects one statistic over my type training efforts. I would like to have a more differentiated view on this. For example I would like to create a profile for my new Piantor keyboard and collect my training efforts/statistics in this profile and it would _pollute_ the statistics for a keyboard I am more proficient with.

What do you think?

**Comment by fehmer** [comment, marked as answer]
you could use tags for this

**Comment by Comradin** [comment]
Thanks, that seems indeed to be what I was looking for. Just did not expect it to be named "tags". 👍 ❤️
